homeassistant/components/intellifire/sensor.py

```diff
@@ -60,7 +60,7 @@ def _downtime_to_timestamp(data: IntellifirePollData) -> datetime | None:
         icon="mdi:fire-circle",
         name="Flame Height",
         state_class=SensorStateClass.MEASUREMENT,
-        value_fn=lambda data: data.flameheight,
+        value_fn=lambda data: (data.flameheight + 1),
```

Comment:
Even though you're the codeowner, it might be a good idea to add a quick comment for future devs re: why the `+ 1` is necessary. 👍🏻